scripts/eslint.py

```diff
@@ -17,6 +17,7 @@
 """Runs ESLint on all relevant JavaScript files in the repository."""
 
 import argparse
+from internal import find_files_for_linting
```

Comment:
The name "internal" sounds a bit too generic for an import here. But I didn't want to rename the folder just because of that, and another option - relative imports - doesn't work (`attempted relative import with no known parent package`).